lib/ui/boxen.ts

```diff
@@ -0,0 +1,39 @@
+import { Options } from 'boxen';
+import { uiLogger } from './logger';
+import { UI_COLORS } from './index';
+import { logger } from '@hubspot/local-dev-lib/logger';
+import { lib } from '../../lang/en';
+
+const defaultOptions: Options = {
+  titleAlignment: 'left',
+  borderColor: UI_COLORS.MARIGOLD,
+  margin: 1,
+  padding: 1,
+  textAlignment: 'left',
+  borderStyle: 'round',
+};
+
+export async function logInBox({
```

Comment:
I figured it made sense to make a wrapper around this so we don't need to repeat the same options and dynamic import in multiple places.